data/json/recipes/food/veggy.json

```diff
@@ -0,0 +1,219 @@
+[
+  {
+    "result": "mushroom_cooked",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 0,
+    "time": 15000,
+    "batch_time_factors": [ 67, 5 ],
+    "autolearn": true,
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [ [ "hotplate", 7 ], [ "char_smoker", 7 ], [ "toolset", 7 ], [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ], [ "aluminum_foil", 5 ] ],
```

Comment:
Aluminium foil or water is a reasonable mechanic. It would be good to abstract this is a `using` set but I'm not sure what you would name it?